packages/components/src/components/brand/brand.scss

```
@@ -18,10 +18,12 @@
 	}
 }
 
+.db-brand-title {
+	/* This is a workaround for internal tests to show svg inside the strong tag */
+	display: contents;
+}
+
```

```suggestion
.db-brand-title {
	/* This is a workaround for internal tests to show svg inside the strong tag */
	display: contents;
}

```

Not necessary anymore, as we won't need the strong tag.